apps/ledger-live-mobile/src/contentCards/cards/hero/elements.tsx

```diff
@@ -47,11 +48,26 @@ export const Tag = ({ label }: LabelProps) => {
   );
 };
 
-export const Title = ({ label }: LabelProps) => {
+export const Title = ({ label, hasSecondaryText }: LabelProps) => {
   const { colors } = useTheme();
 
   return (
-    <Text variant="body" fontWeight="medium" color={colors.neutral.c80} lineHeight={"18px"}>
+    <Text
+      variant="body"
+      fontWeight="medium"
+      color={!hasSecondaryText ? colors.neutral.c80 : undefined}
```

Comment:
color undefined?

Reply:
Oups forgot to push the change should be the default white / black

Reply:
neutral.c100 added